src/main.h

```diff
@@ -202,7 +202,7 @@ std::string GetWarnings(std::string strFor);
 bool GetTransaction(const uint256 &hash, CTransaction &tx, uint256 &hashBlock, bool fAllowSlow = false);
 /** Find the best known block, and make it the tip of the block chain */
 bool ActivateBestChain(CValidationState &state, CBlock *pblock = NULL);
-CAmount GetBlockValue(int nHeight, const CAmount& nFees);
+CAmount GetBlockSubsidy(int nHeight, const Consensus::Params& consensusParams);
```

Comment:
That can still be done as an independent PR. Documentation is good, but there's no reason why it has to be added when a function is touched. For example, should I complete the documentation of all functions touched in  #6163 ? Seems orthogonal to me...
Also I don't see how functions declared main.h should be special in that regard.